ts/@live-compositor/browser-render/src/api.ts

```diff
@@ -0,0 +1,11 @@
+import * as api from './api.generated';
```

Comment:
```suggestion
import * as Api from './api.generated';
```

for consistency with other packages

btw, maybe import it from `live-compositor` directlly instead of copying it here